cli/command/service/create.go

```diff
@@ -58,6 +59,8 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 	flags.Var(&opts.hosts, flagHost, "Set one or more custom host-to-IP mappings (host:ip)")
 	flags.SetAnnotation(flagHost, "version", []string{"1.25"})
 
+	flags.Var(cliopts.NewListOptsRef(&opts.resources.resGenericResources, ValidateSingleGenericResource), "generic-resources", "User defined resources request")
```

Comment:
can you add annotations to the new flags with the API version that's required for this feature? See line 60 above for an example (but check the API version that added this (think 1.35?))

Reply:
Just checked the API is [there since 1.32](https://docs.docker.com/engine/api/v1.32/#operation/TaskInspect) what changed was the CLI

Reply:
Ah, thanks for checking!